iosHackthonApp/Modules/Features/SignUp/Sources/SignUp/SignUpWithEmail.swift

```diff
@@ -12,18 +12,20 @@ import Components
 import shared
 
 struct SignUpWithEmail: View {
-
+	
 	@ObservedObject private var viewModel: SignUpWithEmailViewModel
-
+	
```

Comment:
Just wondering whether we want spaces on blank lines or not?

Reply:
we want a line..no space..fixed

Reply:
Yeah, I'm not sure was just asking as I think I read in the handbook that we should try to avoid trailing spaces and blank line spaces that was all.  Maybe @roger-tan, @Gmiller290488 or @kirstybutler know what is the preferred style?